app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if 'vaccine' not in visitor:
+            raise NotVaccinatedError
+
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
it's better to use `if` and not `elif` because logically few errors can be present at the same time and the interpreter should be able to access all three blocks of code.